internal/cmd/gocdk/_static/demo/secrets.keeper/demo_secrets_keeper.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"html/template"
+	"net/http"
+	"os"
+
+	"gocloud.dev/secrets"
+	_ "gocloud.dev/secrets/localsecrets"
+)
+
+// TODO(rvangent): This file is user-visible, add many comments explaining
+// how it works.
+
+func init() {
+	http.HandleFunc("/demo/secrets.keeper/", secretsKeeperEncryptHandler)
+	http.HandleFunc("/demo/secrets.keeper/encrypt", secretsKeeperEncryptHandler)
+	http.HandleFunc("/demo/secrets.keeper/decrypt", secretsKeeperDecryptHandler)
+}
+
+var keeperURL string
+var keeper *secrets.Keeper
+var keeperErr error
+
+func init() {
+	keeperURL = os.Getenv("SECRETS_KEEPER_URL")
+	if keeperURL == "" {
+		// TODO(rvangent): Remove default later.
+		keeperURL = "base64key://smGbjm71Nxd1Ig5FS0wj9SlbzAIrnolCz9bQQ6uAhl4="
+	}
+	keeper, keeperErr = secrets.OpenKeeper(context.Background(), keeperURL)
+}
+
+type secretsKeeperData struct {
+	URL string
+	Err error
+
+	In     string // user input
+	Out    string // output
+	Base64 bool
+}
+
+const (
+	secretsKeeperTemplatePrefix = `
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>secrets.Keeper demo</title>
+</head>
+<body>
+  <p>
+    This page demonstrates the use of a Go CDK secrets.Keeper.
+  </p>
+  <p>
+    It is currently using a secrets.Keeper based on the URL "{{ .URL }}", which
+    can be configured via the environment variable "SECRETS_KEEPER_URL".
+  </p>
+  <p>
+    See <a href="https://gocloud.dev/concepts/urls/">here</a> for more
+    information about URLs in Go CDK APIs.
+  </p>
+  <ul>
+    <li><a href="./encrypt">Encrypt</a> using the Keeper</li>
+    <li><a href="./decrypt">Decrypt</a> using the Keeper</li>
+  </ul>
+  {{if .Err}}
+    <p><strong>{{ .Err }}</strong></p>
+  {{end}}
+`
+	secretsKeeperTemplateSuffix = `
+</body>
+</html>
+`
+
+	// Input: *secretsKeeperData.
+	secretsKeeperEncryptTemplate = secretsKeeperTemplatePrefix + `
+  <form>
+    Enter plaintext data to encrypt:
+    <br/>
+    <textarea rows="4" cols="50" name="plaintext">{{ .In }}</textarea>
+    <br/>
+    <input type="checkbox" name="base64" value="true" {{if .Base64}}checked{{end}}>
```

Comment:
Let's add an `id` to the checkbox and wrap "The data above is base64 encoded" in a `<label for=<id_value>` tag to make the label text clickable? It's considered more accessible that way because it requires less fine motor control, and it ties the explanation to the input for screen readers.

Reply:
Cool, done.